geometry/frame.hpp

```diff
@@ -16,12 +16,41 @@ namespace internal_frame {
 using base::not_constructible;
 using base::not_null;
 
-template<typename FrameTag, FrameTag frame_tag, bool frame_is_inertial>
+enum Inertia {
+  Inertial,
+  NonInertial,
+};
+
+enum class Handedness {
+  Left,
+  Right,
+};
+
+// The frame is serializable if and only if FrameTag is a protocol buffer enum.
+// To declare a local frame that does not need serialization, use the following
+// pattern:
+//   using MyFrame = Frame<enum class MyFrameTag>;
+//
+// or:
+//   enum class MyFrameTag;
+//   using MyFrame = Frame<MyFrameTag, MyFrameTag{}, Inertial>;
```

Comment:
1. `Frame<enum class MyFrameTag, MyFrameTag{}, Inertial>;` would work (do it that way in the other files);
2. We should put the `tag_` parameter last, so that it can be omitted even if inertia and handedness are provided: when we define a formal namespace-scope serializable frame, having to explicitly specify inertia and handedness is a feature, and for a local throwaway frame with a local enum class, the `tag_` is the most useless parameter (and thus should be the most defaulted one).

Item 2. should probably be done in a separate pull request, it will be a little bit noisy.

Reply:
1. Done.
2. Let's wait until the dust settles.